pom.xml

```diff
@@ -160,6 +160,12 @@
 			<version>1.7.25</version>
 		</dependency>
 
+		<dependency>
+			<groupId>javax.inject</groupId>
+			<artifactId>javax.inject</artifactId>
+			<version>1</version>
+		</dependency>
```

Comment:
I think we can add `<scope>provided</scope>` to this dependency since it will be provided by the JDK or the Java container (Tomcat).